localstack/services/apigateway/invocations.py

```diff
@@ -295,6 +311,14 @@ def invoke_rest_api_integration(invocation_context: ApiInvocationContext):
         invocation_context.response = response
         response = apply_response_parameters(invocation_context)
         return response
+    except ApiGatewayIntegrationError as e:
+        LOG.warning(
+            "Error while invoking integration for ApiGateway ID %s: %s",
+            invocation_context.api_id,
+            e,
+            exc_info=LOG.isEnabledFor(logging.DEBUG),
+        )
+        return e.to_response()
```

Comment:
this is really cool, really like the way we can properly return exceptions and a proper response from it. awesome idea!